setup-utils/install-mxnet-amz-linux.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+######################################################################
+# This script installs MXNet for Python along with all required dependencies on a Amazon Linux Machine.
+######################################################################
+set -e
+# CMake is required for installing dependencies.
+yum install -y cmake
+
+# Set appropriate library path env variables
+echo 'export PATH=/usr/local/bin:$PATH' >> ~/.profile
+echo 'export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH' >> ~/.profile
+echo 'export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH' >> ~/.profile
+echo '. ~/.profile' >> ~/.bashrc
+source ~/.profile
+
+# Install gcc-4.8/make and other development tools on Amazon Linux
+# Reference: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/compile-software.html
+# Install Python, Numpy, Scipy and set up tools.
+yum groupinstall -y "Development Tools"
+yum install -y python27 python27-setuptools python27-tools
+yum install -y python27-numpy python27-scipy python27-nose python27-matplotlib
+
+# Install OpenBLAS at /usr/local/openblas
+git clone https://github.com/xianyi/OpenBLAS
+cd OpenBLAS
+make FC=gfortran -j $(($(nproc) + 1))
+make PREFIX=/usr/local install
+cd ..
+
+# Install OpenCV at /usr/local/opencv
+git clone https://github.com/opencv/opencv
+cd opencv
+mkdir -p build
+cd build
+cmake -D BUILD_opencv_gpu=OFF -D WITH_EIGEN=ON -D WITH_TBB=ON -D WITH_CUDA=OFF -D WITH_1394=OFF -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=/usr/local ..
+make PREFIX=/usr/local install
+
+# Export env variables for pkg config
+export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH
+
+# Install MXNet Core without CUDA
+cd ~/MXNet/mxnet
+cp make/config.mk .
+echo "USE_CUDA=0" >>config.mk
```

Comment:
This is plain bare metal installation. We haven't added support in script to install cuda and mkldnn dependencies. We will upgrade the script later.